src/util/adopt-css.js

```diff
@@ -4,7 +4,7 @@ let style;
  * @param {string} css
  */
 export default function adoptCSS (css, document = globalThis.document) {
-	let window = document.defaultView;
+	let window = document.defaultView ?? document.ownerDocument?.defaultView;
```

Comment:
I don't think `ownerDocument` on `document` makes a lot of sense, wouldn't it always point to itself? Oh, I see, it's a local variable. We should rename it to something else, like `root`.

Reply:
Done